pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/PatientRecordDAOImpl.java

```diff
@@ -84,33 +190,73 @@ public void delete(Long id) {
 		entityManager.flush();
 	}
 
-	
+
 	@Override
 	public PatientRecordDTO getById(Long id) {
-		
+
 		PatientRecordDTO dto = null;
 		PatientRecordEntity pe = this.getEntityById(id);
-		
+
 		if (pe != null){
 			dto = new PatientRecordDTO(pe); 
 		}
 		return dto;
 	}
+	
+	@Override
+	public PatientOrganizationMapDTO getPatientOrgMapById(Long id) {
```

Comment:
Hmm. I guess I was getting confused about what the PateintOrgMap was for. Would the same go for getOrgMapById as well?